api/Document.json

```diff
@@ -5874,13 +5874,60 @@
           }
         }
       },
+      "parseHTML_static": {
+        "__compat": {
+          "description": "`parseHTML` static method",
+          "spec_url": "https://wicg.github.io/sanitizer-api/#dom-document-parsehtml",
+          "tags": [
+            "web-features:parse-html-unsafe",
+            "web-features:html-sanitizer-api"
```

Comment:
I created `web-features:html-sanitizer-api` then discovered that `web-features:parse-html-unsafe` already exists. Should we rename to `html-sanitizer-api` because the API covers both safe and unsafe methods, and both parser and setter methods? 
- If so, how do we rename tags?
- If not, should I just make all of these one entry of "parse-html-unsafe"?

Reply:
Leaving tags alone as suggested by Florian.